pkg/syncthing/config.go

```diff
@@ -0,0 +1,48 @@
+package syncthing
+
+import (
+	"encoding/xml"
+
+	"github.com/syncthing/syncthing/lib/config"
+)
+
+// @source: https://docs.syncthing.net/users/config.html
+func InitConfig() *config.Configuration {
+	return &config.Configuration{
+		Version: 37,
+		GUI: config.GUIConfiguration{
+			Enabled:    true,
+			RawAddress: "0.0.0.0:8384",
+			APIKey:     "envd",
+			Theme:      "default",
+		},
+		Options: config.OptionsConfiguration{
+			GlobalAnnEnabled:     false,
+			LocalAnnEnabled:      false,
+			ReconnectIntervalS:   1,
+			StartBrowser:         true, // TODO: disable later
```

Comment:
fix TODO

Reply:
Ok, I'm only 70% done with the functionality, will do a final clean up later.